src/test/java/org/fungover/mmotodo/team/TeamControllerTest.java

```diff
@@ -0,0 +1,240 @@
+
+        graphQlTester.document(mutation)
+                .execute()
+                .path("updateTeam")
+                .matchesJson(expected);
+    }
+
+
+    @Test
+    void shouldAddNewUserToTeam() throws Exception {
+
+        Mockito.when(teamService.addUserToTeam(1, 1))
+                .thenReturn("user with id 1 successfully added to team");
+
+        String query = "mutation { addUserToTeam(teamId: 1, userId: 1) }";
+        String expected = objectMapper.writeValueAsString("user with id 1 successfully added to team");
+        graphQlTester.document(query)
+                .execute()
+                .path("addUserToTeam")
+                .matchesJson(expected);
+
+    }
+
+    @Test
+    void shouldAddNewTaskToTeam() throws Exception {
+
+        Mockito.when(teamService.addTaskToTeam(1, 1))
+                .thenReturn("task with id 1 successfully added to team");
+
+        String query = "mutation { addTaskToTeam(teamId: 1, taskId: 1) }";
+        String expected = objectMapper.writeValueAsString("task with id 1 successfully added to team");
+        graphQlTester.document(query)
+                .execute()
+                .path("addTaskToTeam")
+                .matchesJson(expected);
+
+    }
+
+    @Test
+    void shouldRemoveUserFromTeam() throws Exception {
+
+        Mockito.when(teamService.removeUserFromTeam(1, 1))
+                .thenReturn("user with id 1 successfully removed from team");
+
+        String query = "mutation { removeUserFromTeam(teamId: 1, userId: 1) }";
+        String expected = objectMapper.writeValueAsString("user with id 1 successfully removed from team");
+        graphQlTester.document(query)
+                .execute()
+                .path("removeUserFromTeam")
+                .matchesJson(expected);
+
+    }
+
+    @Test
+    void shouldRemoveTaskToTeam() throws Exception {
+
+        Mockito.when(teamService.removeTaskFromTeam(1, 1))
+                .thenReturn("task with id 1 successfully removed from team");
+
+        String query = "mutation { removeTaskFromTeam(teamId: 1, taskId: 1) }";
+        String expected = objectMapper.writeValueAsString("task with id 1 successfully removed from team");
+        graphQlTester.document(query)
+                .execute()
+                .path("removeTaskFromTeam")
+                .matchesJson(expected);
+
+    }
+
+    @Test
+    void shouldDeleteTeam() throws Exception {
+        Mockito.when(teamService.deleteTeam(1)).thenReturn("Team with id 1 deleted");
+
+        String query = "mutation { deleteTeam(id: 1) }";
+        String expected = objectMapper.writeValueAsString("Team with id 1 deleted");
+        graphQlTester.document(query)
+                .execute()
+                .path("deleteTeam")
+                .matchesJson(expected);
+
+    }
+
+    private Team createTeam(String name) {
+        Team team = new Team();
+        team.setId(1);
+        team.setName(name);
+        return team;
+    }
+
+
+}
```

Comment:
Good test coverage overall, though missing any tests regarding error handling. There are also some leftover `System.out` that could be removed.